docs/index.rst

```diff
@@ -720,12 +720,8 @@ providing a dict of configuration properties to the instance constructor, e.g.
   consumer = confluent_kafka.Consumer(conf)
 
 
-The supported configuration values are dictated by the underlying
-librdkafka C library. For the full range of configuration properties
-please consult librdkafka's documentation:
```

Comment:
Since it is mostly the librdkafka configuration properties that are important and typically changed, I think it made more sense to have the link to CONFIGURATION.md up front here.

Reply:
I moved it down as people used to read the CONFIGURATION.md before reading the python specific ones. I could find multiple issues where the user was using "log_cb" property instead of "logger" for python client.

Also, I felt that it makes sense to have python client specific properties in the front as it is the python client.

Reply:
Okay, fair enough,.